.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 npm-debug.log
 coverage
 .DS_Store
+package-lock.json
```

Comment:
Haven't wrapped my head around whether we want package-lock.json or yarn.lock to be checked into in the repo. I need to understand the difference in the tree and then figure out which is a better solution for the development team here.